flexget/plugins/sites/newznab.py

```diff
@@ -1,125 +1,615 @@
+    # If you notice you get results of titles that are not in the search query or wrong season/episodes, try this.
+    # NOTE: will most likely break well behaving indexers, so use in a separate task/entry or use nzbhydra
+    newznab:
+        api_server_url: https://api.nzbindexer.com
+        category: tv
+        force_quotes: yes
     """
 
     schema = {
         'type': 'object',
         'properties': {
-            'category': {'type': 'string', 'enum': ['movie', 'tvsearch', 'tv', 'music', 'book']},
-            'url': {'type': 'string', 'format': 'url'},
-            'website': {'type': 'string', 'format': 'url'},
-            'apikey': {'type': 'string'}
+            'api_server_url': {'type': 'string', 'format': 'url'},
+            'api_key': {'type': 'string'},
+            'category': one_or_more({
+                'oneOf': [
+                    {'type': 'integer'},
+                    {'type': 'string', 'enum': list(CATEGORIES)},
+                ]}, unique_items=True),
+            'custom_query': {'type': 'string'},
+            'force_quotes': {'type': 'boolean', 'default': False},
+            'maxage': {
+                'oneOf': [
+                    {'type': 'string', 'format': 'interval'},
+                    {'type': 'string', 'enum': ['auto']}
+                ]
+            },
+            'use_metadata': {
+                'oneOf': [
+                    {'type': 'boolean'},
+                    one_or_more({'type': 'string', 'enum': list(PLUGIN_LOOKUP_MAP)}, unique_items=True)
+                ]
+            },
         },
-        'required': ['category'],
+        'required': ['api_server_url', 'category'],
         'additionalProperties': False
     }
 
-    def build_config(self, config):
-        log.debug(type(config))
+    _movie_identifier_fields = ['movie_name'] + [p.instance.movie_identifier for p in
+                                                 plugin.get_plugins(group='movie_metainfo')]
+    _series_identifier_fields = ['series_name'] + [p.instance.series_identifier for p in
+                                                   plugin.get_plugins(group='series_metainfo')] + ['tvrage_id']
+    _metaid_fields = [p.instance.movie_identifier for p in plugin.get_plugins(group='movie_metainfo')] + \
+                     [p.instance.series_identifier for p in plugin.get_plugins(group='series_metainfo')] + ['tvrage_id']
 
-        if config['category'] == 'tv':
-            config['category'] = 'tvsearch'
-
-        if 'url' not in config:
-            if 'apikey' in config and 'website' in config:
-                params = {
-                    't': config['category'],
-                    'apikey': config['apikey'],
-                    'extended': 1
-                }
-                config['url'] = config['website'] + '/api?' + urlencode(params)
 
+    def prepare_config(self, config):
+        if 'use_metadata' in config:
+            if 'plugins_list' not in config:
```

Comment:
What is `plugins_list`? I don't see it in the schema and why are the two ifs on different lines?